osbrain/agent.py

```diff
@@ -1709,9 +1709,9 @@ def kill(self):
         if self._running:
             self._context.term()
         else:
-            self._die()
+            self._die(linger=0)
 
-    def _die(self):
+    def _die(self, linger):
```

Comment:
Not really necessary, but I think it looks better 😁 

Consider changing the method signature to:
```py
def _die(self, linger=None)
```
And then change the end of `.run()` to `self._die()`

Reply:
Changed signature and renamed attribute.